cask-cli.el

```diff
@@ -87,6 +87,33 @@
           (princ (s-pad-right (+ max cask-cli--table-padding) " " key))
           (princ (concat value "\n")))))))
 
+(defun cask-cli--missing-elpa-dependencies ()
```

Comment:
I don't think we should use _"elpa"_ internally. Everything in Cask is based on elpa and there are no other dependencies than elpa dependencies.